### PR TITLE
optimize: fix wrong usage of Slf4j

### PR DESCRIPTION
**Pull request by chenxi-null**
There are several places that are not correct for the use of Slf4j.

They log error log like this:
```
LOGGER.error("", "send response error", throwable);
```
It will print log like below, missing the log message:
```
[main] ERROR TmpTest -
java.lang.RuntimeException: xx
	at TmpTest.<init>(TmpTest.java:28)
	...... ignore exception stack ......
```

I correct it like this:
```
LOGGER.error("send response error", throwable);
```
So, it will print the expected log content:
```
[main] ERROR TmpTest - send response error
java.lang.RuntimeException: xx
	at TmpTest.<init>(TmpTest.java:28)
	...... ignore exception stack ......
```

### CODE REVIEW

File: core/src/main/java/io/seata/core/rpc/DefaultServerMessageListenerImpl.java [hunk 1]
```diff
@@ -15,31 +15,16 @@
...
-import io.seata.core.protocol.HeartbeatMessage;
-import io.seata.core.protocol.MergeResultMessage;
-import io.seata.core.protocol.MergedWarpMessage;
-import io.seata.core.protocol.RegisterRMRequest;
-import io.seata.core.protocol.RegisterRMResponse;
-import io.seata.core.protocol.RegisterTMRequest;
-import io.seata.core.protocol.RegisterTMResponse;
-import io.seata.core.protocol.RpcMessage;
-import io.seata.core.protocol.Version;
+import io.seata.core.protocol.*;
```
Comment (lightClouds917) [inline core/src/main/java/io/seata/core/rpc/DefaultServerMessageListenerImpl.java]: Do not use import xxx.*

File: core/src/main/java/io/seata/core/rpc/DefaultServerMessageListenerImpl.java [hunk 2]
```diff
@@ -15,31 +15,16 @@
...
 import io.seata.core.rpc.netty.RegisterCheckAuthHandler;
-
-import io.netty.channel.ChannelHandlerContext;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.concurrent.*;
```
Comment (lightClouds917) [inline core/src/main/java/io/seata/core/rpc/DefaultServerMessageListenerImpl.java]: Do not use import xxx.*

File: core/src/main/java/io/seata/core/rpc/netty/AbstractRpcRemotingClient.java [hunk 3]
```diff
@@ -34,19 +24,19 @@
 import io.seata.common.exception.FrameworkException;
 import io.seata.common.thread.NamedThreadFactory;
 import io.seata.common.util.NetUtil;
-import io.seata.core.protocol.AbstractMessage;
-import io.seata.core.protocol.HeartbeatMessage;
-import io.seata.core.protocol.MergeResultMessage;
-import io.seata.core.protocol.MergedWarpMessage;
-import io.seata.core.protocol.MessageFuture;
-import io.seata.core.protocol.RpcMessage;
+import io.seata.core.protocol.*;
```
Comment (lightClouds917) [inline core/src/main/java/io/seata/core/rpc/netty/AbstractRpcRemotingClient.java]: Do not use import xxx.*

File: core/src/main/java/io/seata/core/rpc/netty/AbstractRpcRemotingClient.java [hunk 4]
```diff
@@ -34,19 +24,19 @@
...
 import io.seata.core.rpc.ClientMessageListener;
 import io.seata.core.rpc.ClientMessageSender;
 import io.seata.discovery.loadbalance.LoadBalanceFactory;
 import io.seata.discovery.registry.RegistryFactory;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.net.InetSocketAddress;
+import java.util.List;
+import java.util.concurrent.*;
```
Comment (lightClouds917) [inline core/src/main/java/io/seata/core/rpc/netty/AbstractRpcRemotingClient.java]: Do not use import xxx.*

File: core/src/main/java/io/seata/core/rpc/DefaultServerMessageListenerImpl.java [hunk 5]
```diff
@@ -147,7 +132,7 @@ public void onCheckMessage(RpcMessage request, ChannelHandlerContext ctx, Server
         try {
             sender.sendResponse(request, ctx.channel(), HeartbeatMessage.PONG);
         } catch (Throwable throwable) {
-            LOGGER.error("", "send response error", throwable);
+            LOGGER.error("send response error", throwable);
```
Comment (xingfudeshi) [inline core/src/main/java/io/seata/core/rpc/DefaultServerMessageListenerImpl.java]: please user three parameter method
`LOGGER.error("send response error:{}",throwable.getMessage(), throwable);`

File: core/src/main/java/io/seata/core/rpc/netty/AbstractRpcRemoting.java [hunk 6]
```diff
@@ -427,7 +427,7 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
         try {
             destroyChannel(ctx.channel());
         } catch (Exception e) {
-            LOGGER.error("", "close channel" + ctx.channel() + " fail.", e);
+            LOGGER.error("close channel" + ctx.channel() + " fail.", e);
```
Comment (xingfudeshi) [inline core/src/main/java/io/seata/core/rpc/netty/AbstractRpcRemoting.java]: please user three parameter method

File: core/src/main/java/io/seata/core/rpc/netty/AbstractRpcRemotingClient.java [hunk 7]
```diff
@@ -203,7 +193,7 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) {
                     }
                     sendRequest(ctx.channel(), HeartbeatMessage.PING);
                 } catch (Throwable throwable) {
-                    LOGGER.error("", "send request error", throwable);
+                    LOGGER.error("send request error", throwable);
```
Comment (xingfudeshi) [inline core/src/main/java/io/seata/core/rpc/netty/AbstractRpcRemotingClient.java]: please user three parameter method

File: core/src/main/java/io/seata/core/rpc/netty/AbstractRpcRemotingClient.java [hunk 8]
```diff
@@ -330,7 +320,7 @@ public void run() {
                                 messageFuture.setResultMessage(null);
                             }
                         }
-                        LOGGER.error("", "client merge call failed", e);
+                        LOGGER.error("client merge call failed", e);
```
Comment (xingfudeshi) [inline core/src/main/java/io/seata/core/rpc/netty/AbstractRpcRemotingClient.java]: please user three parameter method

File: core/src/main/java/io/seata/core/rpc/netty/RmMessageListener.java [hunk 9]
```diff
@@ -85,7 +85,7 @@ private void handleBranchRollback(RpcMessage request, String serverAddress,
         try {
             sender.sendResponse(request, serverAddress, resultMessage);
         } catch (Throwable throwable) {
-            LOGGER.error("", "send response error", throwable);
+            LOGGER.error("send response error", throwable);
```
Comment (xingfudeshi) [inline core/src/main/java/io/seata/core/rpc/netty/RmMessageListener.java]: please user three parameter method

File: core/src/main/java/io/seata/core/rpc/netty/RmRpcClient.java [hunk 10]
```diff
@@ -251,7 +251,7 @@ private void sendRegisterMessage(String serverAddress, Channel channel, String d
                     LOGGER.info("remove channel:" + channel);
                 }
             } else {
-                LOGGER.error("", "register failed", e);
+                LOGGER.error("register failed", e);
```
Comment (xingfudeshi) [inline core/src/main/java/io/seata/core/rpc/netty/RmRpcClient.java]: please user three parameter method

File: core/src/main/java/io/seata/core/rpc/DefaultServerMessageListenerImpl.java (hunk 2 above)
Comment (chenxi-null) [reply core/src/main/java/io/seata/core/rpc/DefaultServerMessageListenerImpl.java]: ok, I'll modify it.

File: core/src/main/java/io/seata/core/rpc/DefaultServerMessageListenerImpl.java (hunk 5 above)
Comment (chenxi-null) [reply core/src/main/java/io/seata/core/rpc/DefaultServerMessageListenerImpl.java]: > please user three parameter method
> `LOGGER.error("send response error:{}",throwable.getMessage(), throwable);`

ok, I will fix it.

**Comment by chenxi-null** [comment]
@lightClouds917 @xingfudeshi 
Thanks for your suggestion, I have changed the code to fix the problem that you pointed out.

**Pull request merged**